src/routes/components/components.json

```diff
@@ -2705,5 +2705,151 @@
+		"category": "User Interaction",
+		"tags": ["components and libraries", "images", "typescript"]
+	},
+	{
+		"title": "svelte-resize-observer-action",
+		"url": "https://captaincodeman.github.io/svelte-resize-observer-action/",
+		"repository": "https://github.com/captaincodeman/svelte-resize-observer-action",
+		"description": "Svelte use:action for element resize notifications using ResizeObserver",
+		"npm": "svelte-resize-observer-action",
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": ["components and libraries", "typescript"]
+	},
+	{
+		"title": "svelte-image-comparison",
+		"url": "https://captaincodeman.github.io/svelte-image-comparison",
+		"repository": "https://github.com/captaincodeman/svelte-image-comparison",
+		"description": "Svelte component onion-skin comparison of image or canvas elements",
+		"npm": "svelte-image-comparison",
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": ["components and libraries", "images", "typescript"]
+	},
+	{
+		"title": "svelte-relative-time",
+		"url": "https://captaincodeman.github.io/svelte-relative-time/",
+		"repository": "https://github.com/captaincodeman/svelte-relative-time/",
+		"description": "Tiny Svelte action and Component to render live-updating relative times",
+		"npm": "svelte-relative-time",
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": ["components and libraries", "animations", "typescript"]
+	},
+	{
+		"title": "svelte-signature-pad",
+		"url": "https://captaincodeman.github.io/svelte-signature-pad/",
+		"repository": "https://github.com/captaincodeman/svelte-signature-pad",
+		"description": "Svelte Action to capture smoothed signatures as SVG paths using the perfect-freehand library.",
+		"npm": "svelte-signature-pad",
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": ["components and libraries", "images", "typescript"]
+	},
+	{
+		"title": "svelte-disable-preload",
+		"url": "https://github.com/captaincodeman/svelte-disable-preload",
+		"repository": "https://github.com/captaincodeman/svelte-disable-preload",
+		"description": "Action to block preload event listeners for smoother canvas animations",
+		"npm": "svelte-disable-preload",
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": ["components and libraries", "typescript"]
+	},
+	{
+		"title": "svelte-kit-bot-block",
+		"url": "https://github.com/captaincodeman/svelte-kit-bot-block",
+		"repository": "https://github.com/captaincodeman/svelte-kit-bot-block",
+		"description": "SvelteKit middleware hook to block bots and scripts early",
+		"npm": "svelte-kit-bot-block",
+		"addedOn": "2023-08-16",
+		"category": "",
+		"tags": ["integrations", "typescript"]
+	},
+	{
+		"title": "svelte-intersection-observer-action",
+		"url": "https://captaincodeman.github.io/svelte-intersection-observer-action/",
+		"repository": "https://github.com/captaincodeman/svelte-intersection-observer-action",
```

Comment:
this one has less than 15 stars, so perhaps we should hold off on it https://github.com/svelte-society/sveltesociety.dev/pull/442